### ISSUE TITLE
[bzip2:x64-windows] build failure

### ISSUE THREAD

**Issue by ljere** [microsoft/vcpkg#12597]
**Host Environment**
- OS: Windows 10
- Compiler: Visual Studio 2019 version 16.1

**To Reproduce**
`./vcpkg install bzip2:x64-windows`

**Failure logs** 
C:\Library\vcpkg>vcpkg install bzip2:x64-windows
Computing installation plan...
The following packages will be built and installed:
    bzip2[core]:x64-windows
Detecting compiler hash for triplet x64-windows...
Starting package 1/1: bzip2:x64-windows
Building package bzip2[core]:x64-windows...
Could not locate cached archive: C:\Users\anonymouse\AppData\Local\vcpkg\archives\0e\0ec0bc9db97e4bc77bd00760c2552815b64bfb0c.zip
-- Using cached C:/Library/vcpkg/downloads/bzip2-1.0.6.tar.gz
-- Cleaning sources at C:/Library/vcpkg/buildtrees/bzip2/src/1.0.6-8838f21f8e.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Library/vcpkg/downloads/bzip2-1.0.6.tar.gz
-- Applying patch fix-import-export-macros.patch
-- Applying patch fix-windows-include.patch
-- Using source at C:/Library/vcpkg/buildtrees/bzip2/src/1.0.6-8838f21f8e.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig
-- Acquiring MSYS Packages...
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: C:/Library/vcpkg/downloads/tools/msys2/msys64/usr/bin/bash.exe --noprofile --norc -c "pacman -S --noconfirm --needed pkg-config"
    Working Directory: C:/Library/vcpkg/downloads/tools/msys2
    Error code: 1
    See logs for more information:
      C:\Library\vcpkg\buildtrees\bzip2\msys-pacman-x64-windows-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:127 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:281 (vcpkg_acquire_msys)
  ports/bzip2/portfile.cmake:51 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:79 (include)

Error: Building package bzip2:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: bzip2:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.

**C:\Library\vcpkg\buildtrees\bzip2\msys-pacman-x64-windows-err.log**
```
error: mingw32: key "4A6129F4E4B84AE46ED7F635628F528CF3053E04" is unknown
error: key "4A6129F4E4B84AE46ED7F635628F528CF3053E04" could not be looked up remotely
error: mingw64: key "4A6129F4E4B84AE46ED7F635628F528CF3053E04" is unknown
error: key "4A6129F4E4B84AE46ED7F635628F528CF3053E04" could not be looked up remotely
error: msys: key "4A6129F4E4B84AE46ED7F635628F528CF3053E04" is unknown
error: key "4A6129F4E4B84AE46ED7F635628F528CF3053E04" could not be looked up remotely
error: database 'mingw32' is not valid (invalid or corrupted database (PGP signature))
error: database 'mingw64' is not valid (invalid or corrupted database (PGP signature))
error: database 'msys' is not valid (invalid or corrupted database (PGP signature))
```

**Comment by ljere** [comment]
I fixed it temporarily with https://github.com/microsoft/vcpkg/issues/12567#issuecomment-663807503

**Comment by PhoebeHui** [comment]
@ljere, currently this issue fixed by PR #12467 instead, the PR has been merged, please get latest source and try again.
